docs/guides/build/android.md

```diff
@@ -15,22 +13,19 @@ cargo install cargo-ndk
 rustup target add aarch64-linux-android armv7-linux-androideabi i686-linux-android x86_64-linux-android
 ```
 
-## Generating Apk
-- Go to the [core folder](/core) and `make android`.
-- You can either retrieve the lockbook apk and run it locally via commands or Android Studio
-
-### Commands
-- Using the terminal, enter Lockbook Android's sourcecode at `clients/android`.
-- Run `./gradlew assemble` to generate the APK.
-- The apk will be located at `clients/android/app/build/outputs/apk/debug`.
-
-### Android Studio
-- Download and install Android Studio
-- Using Android Studio, open Lockbook Android at the [android folder](/clients/android).
-- Configure Android Studio to use the SDK installed.
-- Build the APK using the hammer button on the toolbar. Once built, the bottom information
-  bar will give you the option to `locate` the apk.
-  The location of the apk is at `clients/android/app/build/outputs/apk/debug/`.
-- You can also run the APK directly onto your device. You first enable usb debugging on your Android
-  Device, and then connect it to the machine running Android Studio. Then, on the toolbar, you will be given the
-  option to directly run the apk on your device.
+Steps:
+- In `/libs/core_external_interface` run `make android`.
+- Choose one:
+    1. Command Line
+        - In `clients/android` run `./gradlew assemble`.
```

Comment:
I'll let @SmailBarkouch double check, but I'm pretty sure deps are needed for this to work, right?

Reply:
Yes, the dependencies are needed for this command line portion to work.

Reply:
what change do you recommend here? are these dependencies which are not part of the prerequisites list above?